### DISCUSSION TITLE
Reading .gwy files

### DISCUSSION THREAD

**Discussion by nombac** [ContactEngineering/SurfaceTopography#363] [Q&A]
It appears that to obtain the correct orientation, a height array extracted from .gwy files needs to be rotated 90 degrees counterclockwise.

**Comment by pastewka** [comment]
Gwyddion files are read correctly, but MetroPro was flipped. There is now a `show-topography` command line script that allows you to visualize how SurfaceTopography reads and interprets the data. I've changed the visualization to use the same top-left origin that Gwyddion uses.

I will release 1.17.0 that has those fixes in a couple of minutes.
